### ISSUE TITLE
Typo in conversion factor

### ISSUE THREAD

**Issue by danieleongari** [iRASPA/RASPA2#3]
The line of the output 
```
Conversion factor molecules/unit cell -> gr/gr:       x.xxxxxxxxx [-]
```
should read instead:
```
Conversion factor molecules/unit cell -> mg/g:       x.xxxxxxxxx [-]
```
Indeed, in the final section where `Average loading` are reported, this factor is used but with the correct units:
```
Average loading absolute [milligram/gram framework]         xx.xxxxxxx +/-
```

To avoid problems in back compatibility (probably many users already noticed this problem and corrected their parser) I suggest to replace the label to `mg/g` instead of dividing the value by 1000, so that the users are forced to update their parser  instead of suddently have a different value.

**Comment by dubbelda** [comment]
Typo is corrected, thanks for reporting it.

**Comment by yakutovicha** [comment]
Fixed in c1ad4de72c26a50fb08ef3f2ce03703e6fb3a54d
